handlers/handlers_test.go

```diff
@@ -97,11 +99,21 @@ func TestComponentsHandler(t *testing.T) {
 }
 
 func TestDoctorHandler(t *testing.T) {
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+		if err := json.NewEncoder(w).Encode([]byte("")); err != nil {
+			http.Error(w, "error encoding JSON", http.StatusInternalServerError)
+			return
+		}
+	}))
+	defer ts.Close()
+	apiClient, err := config.GetSwaggerClient(ts.URL)
 	doctorHandler := DoctorHandler(
 		mockInstalledComponents{},
 		&mockClusterID{},
 		mockAvailableVersion{},
 		data.NewFakeKubeSecretGetterCreator(nil, nil),
+		apiClient,
 	)
 	resp, err := getTestHandlerResponse(doctorHandler)
```

Comment:
it doesn't start one, the `apiClient` argument there *is* a `*apiclient.WorkflowManager` which has a reference to a running server